documentation/modules/auxiliary/scanner/snmp/snmp_enumshares.md

```diff
@@ -0,0 +1,32 @@
+## Description
+This module will simply scan a range of hosts and queries via SNMP to determine any available shares.
+
```

Comment:
This module has 3 places where it checks: https://github.com/rapid7/metasploit-framework/blob/master/modules/auxiliary/scanner/snmp/snmp_enumshares.rb#L25
It would be good to put those in the docs and explain what they map to